FAQ update for Payment Request (#1088)
btcpayserver/btcpayserver-doc: docs/PaymentRequests.md

![View New Payment Request](./img/payment-requests/NewPaymentRequest.png)

BTCPay creates a URL for the payment request. Share this URL to view your payment request.

> Need mulitple of the same request ? Simply use the Clone option in the main menu for Payment Requests. 

Reviewer: Formatting. 
```suggestion
Need multiple of the same request ? You can use the `Clone` option in the main menu to duplicate payment requests. 
```
